test/parallel/test-http2-session-gc-while-write-scheduled.js

```diff
@@ -14,7 +14,7 @@ const tick = require('../common/tick');
 {
   // This creates a session and schedules a write (for the settings frame).
   let client = http2.connect('http://localhost:80', {
-    createConnection: common.mustCall(() => makeDuplexPair().clientSide)
+    createConnection: common.mustCall(() => new DuplexPair().clientSide)
```

Comment:
Would anyone like to speculate why this test creates a duplex pair and then never uses the other side?